object/src/lib.rs

```diff
@@ -225,6 +225,11 @@ pub enum ReadError {
         #[snafu(backtrace)]
         source: crate::meta::Error,
     },
+    ParseSopAttributes {
```

Comment:
As each error pertains to a single SOP attribute, the singular form (`ParseSopAttribute`) might be a better name.

Don't forget to include a suitable display message!

Reply:
That's a good point. Released an updated version to address these